Library/Homebrew/dev-cmd/test-bot.rb

```diff
@@ -1001,7 +1001,9 @@ def sanitize_output_for_xml(output)
         invalid_xml_pat = /[\x00-\x08\x0B\x0C\x0E-\x1F\uFFFE\uFFFF]/
         output = output.gsub(invalid_xml_pat, "\uFFFD")
       else
-        output = output.delete("\000\a\b\e\f\x2\x1f")
+        # Invalid XML chars, as far as single-byte chars go
+        output = output.delete("\x00\x01\x02\x03\x04\x05\x06\x07\x08\x0b\x0c\x0e\x0f" +
+          "\x10\x11\x12\x13\x14\x15\x16\x17\x18\x19\x1a\x1b\x1c\x1d\x1e\x1f")
```

Comment:
I believe that when splitting a string across multiple lines a continuation using `\` is preferable over using explicit concatenation via `+`. The former should also be a tiny bit more efficient. (To be fair, I haven't checked that.)